modules/oe_content_publication/config/install/field.field.node.oe_publication.oe_publication_last_updated.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.oe_publication_last_updated
+    - node.type.oe_publication
+  module:
+    - datetime
+id: node.oe_publication.oe_publication_last_updated
+field_name: oe_publication_last_updated
+entity_type: node
+bundle: oe_publication
+label: 'Last update date'
+description: ''
+required: false
+translatable: false
+default_value: {  }
```

Comment:
Shall we set the default_value to the current date as we do for the other date fields?

Reply:
No. Last update date is a date of update of the document, not a node.

Reply:
I think this date should be set manually. @ademarco should we set a default value for this field?